completions/tox

```diff
@@ -0,0 +1,26 @@
+# tox(1) completion                                   -*- shell-script -*-
+
+_tox()
+{
+    local cur prev words cword
+    _init_completion || return
+
+    case $prev in
+        -h|--help|--version)
+            return
+            ;;
+        -e)
+            local envs=`tox --listenvs-all`
+            COMPREPLY=( $(compgen -W "${envs}" -- ${cur}) )
+            return 0
+            ;;
+    esac
```

Comment:
Bunch of options are lacking completions, e.g. -c --installpkg, -i, --result-json etc etc. At least, ones that take an argument should have a non-completion installed (like in the --help etc case) if their arguments are not completed.

Reply:
@scop I thought the parse help  will take care of that, so do I need to enlist the no completion for all that don't take an argument?

Reply:
No, help parsing doesn't do anything about option arguments. At least -c, --installpkg, -i, --result-json, --hashseed, --force-dep, and --workdir require an argument, so the completion should either provide argument completions for those as appropriate (like what is currently done for -e), or provide a non-completion for them, which is usually handled by just adding those options to the same case where --help and friends are.